self-samples/src/App.js

```diff
@@ -1,13 +1,19 @@
 import React from 'react';
 import './App.css';
 
-import Sample from './components/Sample';
+import userDetails from "./service";
```

Comment:
Please create a new component in ./components/[yourComponentName]

It should be a name File. It would be very difficult to resolve merge conflicts in this file.